README.rst

```diff
@@ -256,3 +256,63 @@ data that the node is no longer responsible for.
 .. _Jolokia: https://jolokia.org/
 .. _STUPS Cassandra: https://github.com/zalando/stups-cassandra
 .. _Più: http://docs.stups.io/en/latest/components/piu.html
+
+Upgrade your cluster from Cassandra 2.1 -> 3.7
+===================
+
+In order to upgrade your Cluster you should run the following steps.
+
+**Before you actually start, you should read the [Datastax guide](https://docs.datastax.com/en/latest-upgrade/upgrade/cassandra/upgrdCassandraDetails.html) and consider the upgrade restrictions.**
+
```

Comment:
A short intro here defining the outline, like:
"The upgrade is a rolling upgrade and requires you to stop one node at a time", etc...